data/prototypes.json

```diff
@@ -717,7 +717,7 @@
 			},
 			{
 				"john": null,
-				"hashcat": 2810,
+				"hashcat": 2811,
```

Comment:
That's right.

https://github.com/hashcat/hashcat/blob/master/src/modules/module_02811.c